rascal-vscode-extension/tsconfig.json

```diff
@@ -4,7 +4,8 @@
 		"target": "es2020",
 		"outDir": "out",
 		"lib": [
-			"es2020"
+			"es2020",
+			"ES2021.String"
```

Comment:
why do we need this special string lib?

Reply:
Needed for using a specific String method that is no longer part of the code. I removed the line, thanks for seeing it.